sequencing/variant_annotator/dxapp.json

```diff
@@ -91,29 +120,30 @@
       "optional" : true,
       "patterns": [
         "*.tbi"
-      ]
+      ],
+      "help": "(Optional) An array of index files that match the site-only variant files."
     }
   ],
   "runSpec": {
-  "execDepends": [
-      {
-        "name": "pigz"
-      },
-      {
-        "name": "tabix"
+    "distribution": "Ubuntu",
+    "release": "14.04",
+    "interpreter": "python2.7",
+    "file": "src/variant_annotator.py",
+    "timeoutPolicy": {
+      "*": {
+        "hours": 24
       }
-    ],
-    "interpreter": "bash",
-    "file": "src/variant_annotator.sh"
+    }
   },
-  "resources" : "project-BYpFk1Q0pB0xzQY8ZxgJFv1V",
-  "access": {
-    "network": [
-      "*"
-    ],
-     "allProjects": "VIEW"
+  "regionalOptions": {
+    "aws:us-east-1": {"systemRequirements": {"*": {"instanceType": "mem1_ssd1_x8"}},
+                      "assetDepends": [{"id": "record-FJ1pP980FJfZFzqYG88B8Zj1"}]},
+    "azure:westus": {"systemRequirements": {"*": {"instanceType": "azure:mem1_ssd1_x8"}},
+                     "assetDepends": [{"id": "record-FJ1vJgQ99qgf9kQk264zVpKv"}]}
```

Comment:
Please check to use the latest finalized assets when ready.

Reply:
Please update record IDs to:
aws:us-east-1: record-FJ3zf8j08zVZFzqYG88Pbk7z
azure:westus: record-FJ4058j9Vf9ZBBVB2Q7b3VQz

Reply:
Done